src/Collections-Sequenceable-Tests/ArrayTest.class.st

```diff
@@ -602,6 +602,13 @@ ArrayTest >> testAtWrap2 [
 	self assert: (tabTest atWrap: -2) equals: 5
 ]
 
+{ #category : #tests }
+ArrayTest >> testBeReadOnlyLiteral [
+	self assert: (Array new) beReadOnlyLiteral isReadOnlyObject.
```

Comment:
No need parenthese over Array new, String new....

Reply:
I will fix it with the next iteration (which is to improve the sharing code to take the nested arrays into account)